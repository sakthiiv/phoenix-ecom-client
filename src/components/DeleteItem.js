import React from "react";

const DeleteItem = (props) => {
  let { type } = props;
  return <div>Are you sure you want to delete this {type}?</div>;
}

export default DeleteItem;
