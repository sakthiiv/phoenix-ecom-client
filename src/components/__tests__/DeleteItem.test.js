import React from 'react';
import { shallow } from 'enzyme';
import DeleteItem from '../DeleteItem';
describe('Admin', () => {
    it('renders correctly', () => {
      const wrapper = shallow(<DeleteItem />);
      expect(wrapper).toMatchSnapshot();
      // On the first run of this test, Jest will generate a snapshot file automatically.
    });
});