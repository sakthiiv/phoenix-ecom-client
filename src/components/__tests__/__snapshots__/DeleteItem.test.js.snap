// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Admin renders correctly 1`] = `
<div>
  Are you sure you want to delete this 
  ?
</div>
`;
